code/game/machinery/doors/door.dm

```diff
@@ -340,10 +364,35 @@
 	if(!glass && GLOB.cameranet)
 		GLOB.cameranet.updateVisibility(src, 0)
 
+
+/obj/machinery/door/proc/add_lock(var/obj/item/lock_construct/L, mob/user)
+	if(Lock)
+		to_chat(user, "[src] already has [Lock] attached")
+		return
+	else
+		if(user.transferItemToLoc(L, src))
+			user.visible_message("<span class='notice'>[user] adds [L] to [src].</span>", \
+								 "<span class='notice'>You adds [L] to [src].</span>")
+			Lock = L
+
+/obj/machinery/door/proc/check_locked(mob/user)
+	if(Lock)
+		if(Lock.check_locked())
+			to_chat(user, "[src]'s [Lock] is bolted shut")
```

Comment:
Doing a weird thing where the message is returning as "The wooden door's the lock is bolted shut."

Any way to change it to just "lock"?

Reply:
\improper should fix it